h2o-py/h2o/frame.py

```diff
@@ -171,6 +173,10 @@ def get_frame(frame_id):
             return None
         return fr
 
+    def refresh(self):
```

Comment:
Is the idea a user would do self.refresh() to get their frame back to what it was imported as?